internal/clients/team4/testing_utilities.go

```diff
@@ -0,0 +1,56 @@
+package team4
```

Comment:
how is it possible that my changes are in your PR?

Reply:
same with judge_test.go.
Not an issue. Just curious 🤣

Reply:
I'm not sure either lol, I updated both this branch and the custom-client branch with main. Not sure why this is still showing as a difference